yarn-project/telemetry-client/src/otel.ts

```diff
@@ -59,7 +59,7 @@ export class OpenTelemetryClient implements TelemetryClient {
     // https://opentelemetry.io/docs/specs/otel/compatibility/prometheus_and_openmetrics/#resource-attributes
     // https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#supporting-target-metadata-in-both-push-based-and-pull-based-systems
     this.targetInfo = this.meterProvider.getMeter('target').createGauge('target_info', {
-      description: 'Target information',
+      description: 'Target metadata',
```

Comment:
@alexghr I had to make this change or the `metrics` profile from the `docker-compose` template (thanks for setting that up btw!) complained with:

```
otel-collector-1  | 2024-08-22T21:58:16.616Z	error	prometheusexporter@v0.107.0/log.go:23	error gathering metrics: collected metric target_info label:{name:"aztec_network_name" value:"local"} label:{name:"instance" value:"cc74bc34-eda7-48a7-99e1-c7fc76425af9"} label:{name:"job" value:"unknown_service:/home/santiago/.nvm/versions/node/v18.19.0/bin/node"} label:{name:"os_type" value:"linux"} label:{name:"os_version" value:"6.8.0-36-generic"} label:{name:"process_command" value:"/home/santiago/Projects/aztec3-packages-5/yarn-project/aztec/dest/bin/index.js"} label:{name:"process_command_args" value:"[\"/home/santiago/.nvm/versions/node/v18.19.0/bin/node\",\"/home/santiago/Projects/aztec3-packages-5/yarn-project/aztec/dest/bin/index.js\",\"start\",\"--node\",\"--archiver\",\"--sequencer\",\"--pxe\"]"} label:{name:"process_executable_name" value:"/home/santiago/.nvm/versions/node/v18.19.0/bin/node"} label:{name:"process_executable_path" value:"/home/santiago/.nvm/versions/node/v18.19.0/bin/node"} label:{name:"process_owner" value:"santiago"} label:{name:"process_pid" value:"98021"} label:{name:"process_runtime_description" value:"Node.js"} label:{name:"process_runtime_name" value:"nodejs"} label:{name:"process_runtime_version" value:"18.19.0"} label:{name:"service_instance_id" value:"cc74bc34-eda7-48a7-99e1-c7fc76425af9"} gauge:{value:1} has help "Target information" but should have "Target metadata"
otel-collector-1  | 	{"kind": "exporter", "data_type": "metrics", "name": "prometheus"}
otel-collector-1  | github.com/open-telemetry/opentelemetry-collector-contrib/exporter/prometheusexporter.(*promLogger).Println
otel-collector-1  | 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/prometheusexporter@v0.107.0/log.go:23
otel-collector-1  | github.com/prometheus/client_golang/prometheus/promhttp.HandlerForTransactional.func1
otel-collector-1  | 	github.com/prometheus/client_golang@v1.19.1/prometheus/promhttp/http.go:144
otel-collector-1  | net/http.HandlerFunc.ServeHTTP
otel-collector-1  | 	net/http/server.go:2171
otel-collector-1  | net/http.(*ServeMux).ServeHTTP
otel-collector-1  | 	net/http/server.go:2688
otel-collector-1  | go.opentelemetry.io/collector/config/confighttp.(*decompressor).ServeHTTP
otel-collector-1  | 	go.opentelemetry.io/collector/config/confighttp@v0.107.0/compression.go:172
otel-collector-1  | go.opentelemetry.io/collector/config/confighttp.(*ServerConfig).ToServer.maxRequestBodySizeInterceptor.func2
otel-collector-1  | 	go.opentelemetry.io/collector/config/confighttp@v0.107.0/confighttp.go:543
otel-collector-1  | net/http.HandlerFunc.ServeHTTP
otel-collector-1  | 	net/http/server.go:2171
otel-collector-1  | go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp.(*middleware).serveHTTP
otel-collector-1  | 	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp@v0.53.0/handler.go:218
otel-collector-1  | go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp.NewMiddleware.func1.1
otel-collector-1  | 	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp@v0.53.0/handler.go:74
otel-collector-1  | net/http.HandlerFunc.ServeHTTP
otel-collector-1  | 	net/http/server.go:2171
otel-collector-1  | go.opentelemetry.io/collector/config/confighttp.(*clientInfoHandler).ServeHTTP
otel-collector-1  | 	go.opentelemetry.io/collector/config/confighttp@v0.107.0/clientinfohandler.go:26
otel-collector-1  | net/http.serverHandler.ServeHTTP
otel-collector-1  | 	net/http/server.go:3142
otel-collector-1  | net/http.(*conn).serve
otel-collector-1  | 	net/http/server.go:2044
```